jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/DeleteObjectAction.kt

```diff
@@ -29,35 +29,30 @@ class DeleteObjectAction : S3ObjectAction(message("s3.delete.object.action"), Al
 
     // TODO enable for versioned objects.
     override fun enabled(nodes: List<S3TreeNode>): Boolean = nodes.isNotEmpty() && nodes.all { it::class == S3TreeObjectNode::class }
-}
-
-fun deleteSelectedObjects(project: Project, treeTable: S3TreeTable) {
-    val nodes = treeTable.getSelectedNodes().filterIsInstance<S3TreeObjectNode>()
-    deleteNodes(project, treeTable, nodes)
-}
 
-private fun deleteNodes(project: Project, treeTable: S3TreeTable, nodes: List<S3TreeObjectNode>) {
-    val response = Messages.showOkCancelDialog(
-        project,
-        message("s3.delete.object.description", nodes.size),
-        message("s3.delete.object.action"),
-        message("general.delete"),
-        message("s3.delete.object.cancel"),
-        Messages.getWarningIcon()
-    )
+    private fun deleteNodes(project: Project, treeTable: S3TreeTable, nodes: List<S3TreeObjectNode>) {
```

Comment:
This function is used in multiple places, moving it into the class breaks that which will break tests and runtime

Reply:
It shouldn't, its not used anywhere else now that we handle delete through the shortcut isntead of key listener

Reply:
Cool, I wasn't sure exactly where else it was used.